src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DomainManagedDeploymentsTestCase.java

```diff
@@ -57,7 +62,8 @@ public static void cleanUp() {
 
     @Before
     public void before() {
-        browser.navigate().refresh();
+        Graphene.goTo(HomePage.class);
```

Comment:
You can use Console.withBrowser(browser).refreshAndNavigate method to minimize redundancy in @before